packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -266,16 +285,28 @@ export default class TelecomPackMigrationOffersCtrl {
             ),
           }));
 
-        if (gtr.length > 0) {
+        if (this.isGrt && this.currentOfferTechno === offerTechno) {
+          this.grtLabel = `telecom_pack_migration_${gtr[0].name}`;
+          this.grtLabelPrice = gtr[0].optionalPrice.text;
+
+          // Initialize GTR selected to first value
+          set(offer, 'gtrSelected', gtr[0].name);
```

Comment:
```suggestion
          offer.gtrSelected = gtr[0].name
```

Reply:
No we can't do that, we are into a map to add some elements need to complete data to display into offers list.